### ISSUE TITLE
Built error: The getter 'accentColor' isn't defined for the class 'ThemeData'.

### ISSUE THREAD

**Issue by LiveRock** [RatelHub/rflutter_alert#138]
After upgrade Flutter to 
Flutter (Channel stable, 3.10.0, on macOS 13.3.1 22E772610a darwin-arm64, locale en-GB)

rflutter_alert-2.0.4/lib/src/dialog_button.dart:63:43: Error: The getter 'accentColor' isn't defined for the class 'ThemeData'.

**Comment by bradintheusa** [comment]
This will work while we wait for the PR to be merged
```yaml

  rflutter_alert: 
    git: https://github.com/kent2508/rflutter_alert
```

**Comment by LiveRock** [comment]
Thanks, it works.
Will wait for official release.

**Comment by ibrahimdevs** [comment]
@LiveRock it is fixed with V2.0.5, you can upgrade with:

`  rflutter_alert: ^2.0.5`

**Comment by LiveRock** [comment]
Thanks, it works
